config-model/src/test/java/com/yahoo/vespa/model/container/component/ModelTest.java

```diff
@@ -0,0 +1,36 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.model.container.component;
+
+import com.yahoo.config.model.deploy.DeployState;
+import com.yahoo.text.XML;
+import org.junit.jupiter.api.Test;
+
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * @author hmusum
+ */
+public class ModelTest {
+
+    @Test
+    void invalid_url(){
+        var xml = """
+                <component id="bert-embedder" type="bert-embedder">
+                  <transformer-model url="models/e5-base-v2.onnx" />
+                  <tokenizer-vocab path="models/vocab.txt"/>
+                </component>
+                """;
+
+        try {
```

Comment:
Consider simplifying by using Junit5's `assertThrows` instead of the try-catch-fail pattern.

Reply:
Good idea, done